.travis.yml

```diff
@@ -24,7 +24,7 @@ python: 3.7
 
 before_install:
   - pip install pycodestyle
-  - pycodestyle --exclude=".git,.tox,__pycache,venv,.eggs,build,jupyterhub_config.py"
+  - pycodestyle --exclude=".git,.tox,__pycache,venv,.eggs,build,jupyterhub_config.py,cylc/uiserver/websockets/"
```

Comment:
This is due to the `graphene-tornado` class that we copied from an open PR. Mainly due to line number that is greater than 80 there :+1: so we need to ignore that for now.